src/llm_models/openai.rs

```diff
@@ -440,6 +440,14 @@ impl LLMModel for OpenAIModels {
             },
         }
     }
+
+    // Accepts a [0-100] percentage range and returns the target temperature based on model ranges
+    fn get_normalized_temperature(&self, relative_temp: u32) -> f32 {
+        // Temperature range documentation: https://platform.openai.com/docs/api-reference/chat/create
+        let min = 0u32;
+        let max = 2u32;
```

Comment:
OpenAI is the only one I found to have a 0-2 range, hence a model-specific implementaiton.